code/modules/fields/timestop.dm

```diff
@@ -85,6 +86,8 @@
 		freeze_projectile(A)
 	else if(istype(A, /obj/mecha))
 		freeze_mecha(A)
+	else if((ismachinery(A) && !istype(A, /obj/machinery/light)) || isstructure(A)) //Special exception for light fixtures since recoloring causes them to change light
+		freeze_structure(A)

```

Comment:
What about make list of ignored types instead of hardcode `machinery/light` ignoring?

Reply:
Doable if necessary, but right now there's only one type to check for, so I kept it simple.